src/redux/configureStore.js

```diff
@@ -8,8 +10,9 @@ const reducers = combineReducers({
   categories: categoriesReducer,
 });
 
-const store = configureStore({
-  reducer: reducers,
-});
+const store = configureStore(
+  { reducer: reducers },
+  applyMiddleware(thunk),
```

Comment:
- [ ] [optional] @Danie12345 great work, it would also be nice to add a redux logger, it would assist in debugging and showing your dataflow.